src/firefly/js/visualize/ui/MatchLockDropDown.jsx

```diff
@@ -10,10 +10,40 @@ import {DropDownToolbarButton} from '../../ui/DropDownToolbarButton.jsx';
 import MATCH_LOCKED from 'html/images/28x28_Match_Locked.png';
 import MATCH_UNLOCKED from 'html/images/28x28_Match_Unlocked.png';
 import {isImage} from '../WebPlot';
+import {showInfoPopup} from 'firefly/ui/PopupUtil';
 
 function changeMatchType(vr, matchType, lockMatch) {
     const plot= primePlot(vr);
     if (!plot) return;
+    //check for images without WCS when aligning by WCS or Target, and display appropriate warning
+    if(matchType.key === 'Standard' || matchType.key === 'Target') {
+        const plotViewObjs = getPlotViewAry(vr);
+        const warningTitles = [];
+        for (const i in plotViewObjs) {
+            const currPlots = plotViewObjs[i].plots;
+            for (const j in currPlots) {
+                const plt = currPlots[j];
+                const checkWCSExists = hasWCSProjection(plt);
+                if (!checkWCSExists) {
+                    if (!warningTitles.includes(plt.title)) {
+                        warningTitles.push(plt.title);
+                    }
+                }
+            }
+        }
+
+        if (warningTitles.length !== 0) { //at least one image without WCS
+            const msgTitle = 'The following image(s) do not have WCS:';
+            const msgDesc = (String(warningTitles)).replace(/,/g, ', '); //warningTitles.join('\r\n');
+            const renderContent = <div>
+                    {msgTitle} <br></br> <br></br>
+                    <div style={{padding: '5px 5px 5px 5px', whiteSpace: 'normal', letterSpacing: '1px', lineHeight: '1.5'}}>
+                        {msgDesc}
+                    </div>
+            </div>;
+            showInfoPopup(renderContent, 'Warning');
```

Comment:
`eslint` should have flagged this.  Multiline jsx should be wrapped in parentheses.  i.e.
```
        const renderContent = (
            <div>
                    {msgTitle} <br></br> <br></br>
                    <div style={{padding: '5px 5px 5px 5px', whiteSpace: 'normal', letterSpacing: '1px', lineHeight: '1.5'}}>
                        {msgDesc}
                    </div>
            </div>);
  ```
  `{padding: '5px 5px 5px 5px'` can be  `{padding: '5px'}`.  Or `{padding: 5}`, React will auto add the px.

Reply:
Got it, will make this change. eslint didn't flag this, and it also imports using double quotes for me. So I'll need to look at its settings again.